pkg/tests/end_to_end_tests/main_test.go

```diff
@@ -126,7 +134,7 @@ func TestMain(m *testing.M) {
 				os.Exit(1)
 			}
 
-			storagePath, err := generatePrometheusWALFile()
+			storagePath, err := generatePrometheusWAL()
```

Comment:
how come the old function `generatePrometheusWALFile` was not deleted?

Reply:
also why did the old function need to change?

Reply:
The old function takes a snapshot. Taking a snapshot creates a block, removing all exemplars from the WAL. But, this can be simplified to a single function, as done in recent commits.